ci-operator/config/opendatahub-io/notebooks/opendatahub-io-notebooks-main.yaml

```diff
@@ -58,6 +58,33 @@ images:
 - context_dir: .
   dockerfile_path: runtimes/rocm-tensorflow/ubi9-python-3.11/Dockerfile.rocm
   to: rocm-runtime-tensorflow-ubi9-python-3.11
+- context_dir: .
+  dockerfile_path: jupyter/minimal/ubi9-python-3.12/Dockerfile.cpu
+  to: jupyter-minimal-ubi9-python-3.12
+- context_dir: .
+  dockerfile_path: jupyter/datascience/ubi9-python-3.12/Dockerfile.cpu
+  to: jupyter-datascience-ubi9-python-3.12
+- context_dir: .
+  dockerfile_path: jupyter/minimal/ubi9-python-3.12/Dockerfile.cuda
+  to: cuda-jupyter-minimal-ubi9-python-3.12
+- context_dir: .
+  dockerfile_path: jupyter/tensorflow/ubi9-python-3.12/Dockerfile.cuda
+  to: cuda-jupyter-tensorflow-ubi9-python-3.12
+- context_dir: .
+  dockerfile_path: jupyter/pytorch/ubi9-python-3.12/Dockerfile.cuda
+  to: cuda-jupyter-pytorch-ubi9-python-3.12
+- context_dir: .
+  dockerfile_path: jupyter/trustyai/ubi9-python-3.12/Dockerfile.cpu
+  to: jupyter-trustyai-ubi9-python-3.12
+- context_dir: .
+  dockerfile_path: jupyter/minimal/ubi9-python-3.12/Dockerfile.rocm
+  to: rocm-jupyter-minimal-ubi9-python-3.12
+- context_dir: .
+  dockerfile_path: jupyter/rocm/tensorflow/ubi9-python-3.12/Dockerfile.rocm
+  to: rocm-jupyter-tensorflow-ubi9-python-3.12
+- context_dir: .
+  dockerfile_path: jupyter/rocm/pytorch/ubi9-python-3.12/Dockerfile.rocm
+  to: rocm-jupyter-pytorch-ubi9-python-3.12
```

Comment:
I think I see the issue from the rehearsals. It looks like at this step, we don't specify where the CodeServer and Runtimes Dockerfiles.
@Hannah-Tosi, could you please update this section to include the missing images? Also, please double-check the other sections to ensure everything is aligned.